force-app/main/default/classes/EmailUtility.cls

```diff
@@ -0,0 +1,36 @@
+public class EmailUtility {
+    public static void toSendEmailWithTemplate(String Name, String objectType){
+        List<Messaging.SingleEmailMessage> mailList = new List<Messaging.SingleEmailMessage>();
+        
+        EmailTemplate emailTemp = [SELECT Id,
+                                  Subject,
+                                  Body
+                                  FROM EmailTemplate
+                                  WHERE DeveloperName = 'Decision_Maker'];
+        
+        try{
```

Comment:
Follow code format and line spaces\tabs

try {
....//some code
....//some code
}	
	
instead of
try {
//some code
//some code
}

review and update for this try block